app/Intercept_Components/InterceptTextBox.tsx

```diff
@@ -1,25 +1,51 @@
 import * as React from 'react'
 
 export const InterceptTextBox = props => {
-  console.log(props)
   let responseText = "{msg:hello}";
+  const checkResponseExists = (index) => {
+    if(props.ResponseText[index]){
```

Comment:
Why are we still using `index`? we already had discussion and decided to go with request id.

Reply:
I have used requestId as you said and that is what is passed in the call. Will change the names and push the changes